conf/application.conf

```diff
@@ -75,7 +75,7 @@ microservice {
     }
     vat-api {
       host = localhost
-      port = 9154
+      port = 9156
```

Comment:
I see what you mean about this. It's not even being used as its the "off" case for an old feature switch that has been turned on for ages. I think for now this is fine and we can remove this `vat-api` config as part of that feature switch removal down the line?